src/style.rs

```diff
@@ -51,11 +82,16 @@ impl Default for AlignContent {
     }
 }
 
+/// Sets the layout used for the children of this node
+///
+/// [`Display::Flex`] is the default value.
 #[derive(Copy, Clone, PartialEq, Eq, Debug)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum Display {
```

Comment:
This definition does not actually exist in the specification. As far as I can tell, these are two separate properties: 

- Inner Display Layout Model: `display-inside`
https://www.w3.org/TR/css-display-3/#box-generation
Which defines what layout model to use (flexbox vs css-grid vs flow vs table ++)
- Box Generation: `display-box`
https://www.w3.org/TR/css-display-3/#valdef-display-none
Which defines if the flex-item will generate any boxes at all

So, having this a one enum currently works and is probably convenient, however it does not feel _techically precise_

I think I'd prefer creating the two separate distinct parameters, when wrap them in a third convenience wrapper if possible, but it is not something that needs to be done asap

Thoughts?

Reply:
Make an issue and we'll do it for 0.2